sql/voters.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var VotersSql = {
+	sortFields: [
+		'username',
+		'address',
+		'publicKey'
+	],
+
+	getVoters: 'SELECT ARRAY_AGG("accountId") AS "accountIds" FROM mem_accounts2delegates WHERE "dependentId" = ${publicKey}'
```

Comment:
Why we are not using `json-sql` here?

Reply:
The SQL query is inherited from delegates module. Usages of `json-sql` and `pg-promise` should be unified across the whole codebase but it is not a part of this task I think